setup.py

```diff
@@ -11,7 +11,8 @@
     author_email='jarbasai@mailfence.com',
     license='Apache-2.0',
     packages=['neon_stt_plugin_scribosermo'],
-    install_requires=["ds-ctcdecoder", "tflit",
+    install_requires=["ds-ctcdecoder",
```

Comment:
This should implement `get_requirements` like [here](https://github.com/NeonGeckoCom/NeonCore/blob/abffb2cfccba9f527945505778268e9dc803d5e6/setup.py#L44) for consistency with other Neon packages